assets/js/translator.js

```diff
@@ -558,6 +564,25 @@ function populateTranslationList() {
     }
 }
 
+function getSrcNumCols(numSrcCols) {
```

Comment:
Also I wonder if we could combine these by detecting which direction the dropdown opens up in and performing accordingly. Bootstrap JS might be setting some class that could be useful in detecting that... Not really important though. Are we currently forcing them to open up in the directions we want them to? If not, we definitely should. I think there's a Bootstrap `data-*` setting for it?